index.rst

```diff
@@ -44,14 +44,465 @@
+
+The standardized use-cases associated with this deliverable are found at :ref:`pg-D1-Use-cases` page.
+
+.. _Deliverable 2:
+
+Deliverable 2:
+--------------
+
+.. note:: 
+
+   The deliverable description from the `charge`_ is in the note below to ease readability 
+
+   2. A set of requirements to augment, clarify, or remove ambiguity that may occur from looking only at the use-cases.
+      
+      - This is not a formal and all-encompassing LSE-type requirements document.
+
+Like the use-cases, the requirement deliverables have been separated into two different categories, those which are focused on tooling and procedures for on-the-fly analysis and image visualization.
+The two categories are quite separate in nature but not completely independent.
+
+The detailed list of requirements is found on :ref:`pg-D2-Requirements`.
+
+
+.. _Deliverable 3:
+
+Deliverable 3:
+--------------
+
+.. note:: 
+
+   The deliverable description from the `charge`_ is in the note below to ease readability 
+
+   1. A summary of the suite of currently available tools/systems that may help fulfil the required functionality.
+
+      - Report on their status, limitations, and where their current requirements may need to be expanded to cover any increase in scope
+      - Tools which were considered but found to be inadequate must also be reported
+
+This section contains a summary of the tools currently available for use that are applicable to on-the-fly operations.
+What follows is a list of the tools, each with a short description that contain links to pertinent information.
+
+A high-level overview of these tools and computering resources was presented at the `original presentation given to stakeholders <https://docs.google.com/presentation/d/1i4p-sg42FXtEqGVqIZMeFadWSZZ0Lu_CpoqEafkMfy4/edit#slide=id.gd8dafc0d0d_0_30>`_.
+These slides may contain additional information and/or a more detailed description and use-case.
+
+Currently Available Tools
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following tools are released for general use and are expected to be used in the context of on-the-fly information gathering and display.
+Some remain under active development whereas others are considered nearly complete.
+
+LOVE
+''''
+
+.. tiago writing this
+
+Nublado (Jupyter Notebook) Interface
+''''''''''''''''''''''''''''''''''''
+
+.. patrick writing this
+
+Camera Visualization Tool
+''''''''''''''''''''''''''
+
+.. tony writing this
+
+EFD and LFA
+'''''''''''
+
+.. Patrick writing this
+
+Chronograf
+''''''''''
+
+.. Patrick writing this
+
+Watcher
+'''''''
+
+.. Patrick writing this
+
+SAL Scripts
+'''''''''''
+
+.. Patrick writing this
+
+OCPS
+'''''''
+
+.. Patrick writing this
+
+
+Prompt Processing Data Products
+'''''''''''''''''''''''''''''''
+
+and delivery `via the Telemetry Gateway <https://docushare.lsst.org/docushare/dsweb/Get/LSE-72#%5B%7B%22num%22%3A54%2C%22gen%22%3A0%7D%2C%7B%22name%22%3A%22XYZ%22%7D%2C69%2C205%2C0%5D>`_.
+
+.. Patrick writing this
+
+faro
```

Comment:
Link to documentation:
https://pipelines.lsst.io/v/daily/modules/lsst.faro/index.html

Link to github repo:
https://github.com/lsst/faro

faro is a framework for automatically and efficiently computing scientific performance metrics on the outputs of the LSST Science Pipelines for units of data of varying granularity, ranging from single-detector to full-survey summary statistics, and persists the results as scalar metric values alongside the input data products in the same butler repo. In the "first-look analysis" context, it is intended that faro would be run as an afterburner to OCPS and prompt processing (run automatically as part of the same pipeline) and would compute scalar metrics to quantify the performance of individual visits as they are acquired. For example, faro could be used to estimate the effective depth of individual images by computing the flux of a point source that would be measured with signal-to-noise of 10, or to measure variations in effective depth across the focal plane that would be indicative of variable atmosphere transparency. The intent is that these metrics would be available within minutes to the observers to inform nighttime operations. faro is designed to be modular and configurable so that additional metrics can be readily added to support summit operations.

faro is NOT itself a visualization tool, but rather generates scalar metric values that could be used as input to visualization tools.

Reply:
Thanks! content has been incorporated.
I didn't add the github repo link as I didn't find a natural place for it. For the moment I've left it as a comment in the file.